extract.py

```diff
@@ -162,7 +162,7 @@ def _ignore_text(content, style):
 
     rgb_color = get_dict_nested_value(style, 'foregroundColor', 'opaqueColor', 'rgbColor')
 
-    if not rgb_color or len(rgb_color) == 0:
+    if not rgb_color or len(rgb_color) == 0 or rgb_color == {'red': 0.13333334, 'blue': 0.13333334, 'green': 0.13333334}:
```

Comment:
So this is the error we were encountering regarding block_titles, labels and question_titles, it is to do with the type of black colour used in the google slides, previously this condition not being here meant these specifics of each block were being ignored, hence not showing up in the block files